platform-common-string/src/test/java/com/softicar/platform/common/string/csv/CsvTokenizerTest.java

```diff
@@ -0,0 +1,285 @@
+package com.softicar.platform.common.string.csv;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CsvTokenizerTest extends Assert {
```

Comment:
there should be a test for quotes in unquoted values

Reply:
added `testWithQuoteInNonQuotedValue`